app/build.gradle

```diff
@@ -72,6 +72,9 @@ dependencies {
     compile "com.android.support:design:${SUPPORT_LIB_VERSION}"
     compile 'com.squareup.picasso:picasso:2.5.2'
     compile 'com.squareup:otto:1.3.8'
+    compile('com.github.afollestad.material-dialogs:core:0.8.5.3@aar') {
```

Comment:
I mean the underlined highlighted texts such which you can see with yellow color in this pic(these are URl's) were not visually like this (they were very light in color ) then I use (.linkColorAttr method provided by this library) to get desired result.Also making further changes to dialogue is comparatively easy with material dialogues(used by me) the appcompat.